migration/sql-files/002-tracker-items.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE ONLY tracker_items
```

Comment:
(we're currently not testing this in a good way, but still, doesn't change the fact)